cmd/test-utils/utils.go

```diff
@@ -1,4 +1,4 @@
-package ops
+package testutils
```

Comment:
@aayushKumarJarvis Can we make the directory name also as `testutils` instead of `test-utils`?